01-numpy.md

```diff
@@ -505,19 +505,35 @@ matplotlib.pyplot.show(image)
 Blue regions in this heat map are low values, while red shows high values.
 As we can see,
 inflammation rises and falls over a 40-day period.
+
+> ## Some IPython magic {.callout}
+>
+> If you're using an IPython / Jupyter notebook,
+> you'll need to execute the following command
+> in order for your matplotlib images to appear
```

Comment:
To appear _in_ the notebook, because there's generally nothing stopping them from appearing otherwise (unless you're over SSH or something).